CPE200Lab1/CPE200Lab1/Form1.cs

```diff
@@ -12,9 +12,106 @@ namespace CPE200Lab1
 {
     public partial class Form1 : Form
     {
+        float  value = 0, value2=0;
+        string previousoper = "+";
```

Comment:
Variable name is hard to read

previousOper  <- Camel case
previous_oper  <- Snake case